gnovm/pkg/gnolang/machine.go

```diff
@@ -2231,6 +2232,17 @@ func (m *Machine) String() string {
 	return builder.String()
 }
 
+func (m *Machine) ExceptionsStacktrace() string {
+	var builder strings.Builder
+
+	for _, ex := range m.Exceptions {
+		builder.WriteString(fmt.Sprintf("panic %s\n", ex.Sprint(m)))
+		builder.WriteString(fmt.Sprintf("%s", ex.Stacktrace.String()))
```

Comment:
If you have a `defer func() { panic("hello") }`, for instance in the recursive function of `panic2.gno`, this would also generate a hige list of panics and associated stacktraces. (This might be a case you want to test, btw)

I think you can probably get away with only printing the first and the last, and then having "... %d panics elided ...".

BTW ideally this shouldn't be the case - we should have one stacktrace for the panic, with the deferred functions at the top, then the panic location, then the rest of the trace. But I understand why you did this - our way to unwind the stack means that if you want information of when a panic occurred, you need to get it at the time of the panic itself.

Reply:
Yes right, I miss this one. 

Done here: https://github.com/gnolang/gno/pull/2145/commits/4ee0eeb80a97cf2a6010cc18b3b94889979845ec